libs/sqlserver/tests/integration_tests/test_vectorstores.py

```diff
@@ -868,55 +869,56 @@ def test_sqlserver_batch_add_documents(
 
 
 def test_sqlserver_batch_add_documents_with_batch_size(
-    store: SQLServer_VectorStore,
     texts: List[str],
 ) -> None:
-    """Test that `add_documents` returns equivalent number of ids of input
-    texts when using more than 500 documents and custom batch_size."""
+    """Test that when store is re-initialized with a different batch_size,
+    the new value is not accepted by the store object. In below case we should not get
+    an error even with a new batch_size(900) > 419, because that value cannot be
+    assigned after instantiating the object."""
 
     text_splitter = RecursiveCharacterTextSplitter(chunk_size=3, chunk_overlap=1)
     split_documents = text_splitter.create_documents(texts)
-    store._batch_size = 400
+    store = connect_to_vector_store(_CONNECTION_STRING_WITH_UID_AND_PWD, 230)
+    store._batch_size = 900
```

Comment:
In C++, this would have been a compile error. I know Python is an interpreted language, no compilation - so wouldnt this end up being a runtime error since _batch_size shouldnt be accessible?

Reply:
Or is the private member named `__batch_size` and since this is _batch_size` notice the difference in name, does it mean we are setting a new variable altogether?